pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -221,5 +274,8 @@ spec:
     - refName: metrics-server*
     - refName: secretgen-controller*
     - refName: pinniped*
+    - refName: tkg-storageclass*
+      valuesFrom:
```

Comment:
@vijaykatam This additional package is only defined in a clusterbootstrap, but not clusterbotstraptemplate, as it will generate secret name and content according to a few data values. Is this a best practice?

Reply:
This is fine, but when users set static secrets we expect them to manage their lifecycle via explict ownerRef's because addons-manager does not know if that secret is in use elsewhere. 

Also consider using `inline` option if there is no sensitive data in the secret as this removes the need to set ownerRef's